commons/src/test/java/gov/cms/qpp/conversion/ErrorCodeTest.java

```diff
@@ -0,0 +1,51 @@
+package gov.cms.qpp.conversion;
+
+import gov.cms.qpp.conversion.model.error.ErrorCode;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import com.google.common.truth.Truth;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+class ErrorCodeTest {
```

Comment:
We already have an existing `ErrorCodeTest`.  `gov.cms.qpp.conversion.model.error.ErrorCodeTest`.  Looks like it wasn't moved over when `ErrorCode` was moved to `common` or something.  We'll need to integrate it.